cli/flox/src/commands/activate.rs

```diff
@@ -175,6 +175,13 @@ impl Activate {
             other => other?,
         };
 
+        // Must come after getting an activation path to prevent premature
+        // locking or migration.
+        subcommand_metric!(
+            "activate::version",
```

Comment:
nit: i'm not sure we should use `::` for both "metadata" (here) and subcommands (e.g. service commands)

Reply:
perhaps structuring the event slightly differently, 
```rust
subcommand_metric!(
   "activate",
   metadata = "lockfile_version",
   value = environment.lockfile(&flox)?.version(),
)
```

or just leave out the namespace (i think that's how we do that with failed installs:

https://github.com/flox/flox/blob/5aa43f1ee5709635860b71e54315093f4a6eb5b5/cli/flox/src/commands/install.rs#L223-L226

Reply:
Wouldn't that cause us to double-count the `activate` metric in Databox?

Bearing in mind that we have these constraints:

> This is done in a second `subcommand_metric!`, in addition to the early
> call in `Activate::handle`, because we don't have all of the information
> required at the start, don't want to cause a premature lock/error, and
> our in-house metrics don't have a way to additional submit tags on
> existing metrics.

Reply:
if you only count `event == "activate"` then yes.

if we do want it to be a different event it could be just a different separator "activate#lockfile_version"

Reply:
> if you only count `event == "activate"` then yes.

That seems more confusing that needing to differentiate between metadata / commands by metric name.

> if we do want it to be a different event it could be just a different separator "activate#lockfile_version"

Yeah, that could work. Though I'd still favour `activate#version` with a field of `lockfile_version` so that we have the space to add `manifest_version` to the same event later.

The other option is that we just submit `activate` later in the function, once we've passed some of the initial validation and have a lock, on the basis that we don't have a successful command before then. Thoughts?

Reply:
The `activate#version` solution seems pretty good to me. Let's do that and move forward.